zone/spell_effects.cpp

```diff
@@ -1820,6 +1820,63 @@ bool Mob::SpellEffect(Mob* caster, uint16 spell_id, float partial, int level_ove
 
 				break;
 			}
+			case SE_SummonCorpseZone:
+			{
+#ifdef SPELL_EFFECT_SPAM
```

Comment:
Can we remove this? If we're going to utilize logging, please just use `LogSpells("Message")`

Reply:
That's just how we do spell effect shit, I think it's fine for now. Maybe we can switch them all over to LogSpells at another time :P